src/components/Icon/assets/iconSelector.ts

```diff
@@ -61,6 +62,7 @@ const iconSelector = {
   checkedList: CheckedListIcon,
   hourglass: HourglassIcon,
   ticTacToeArrow: TicTacToeArrowIcon,
```

Comment:
At some point we need to ask the UX team for proper names on the icons.

Reply:
i dont think its their job exactly but we need to have the same names for reference